dts/arm/st/l1/stm32l152.dtsi

```diff
@@ -1,11 +1,7 @@
 /*
- * Copyright (c) 2019 Antony Pavlov <antonynpavlov@gmail.com>
+ * Copyright (c) 2020 Abel Sensors
  *
  * SPDX-License-Identifier: Apache-2.0
  */
 
-/*
- * The STM32L151 and STM32L152 differ in that
- * the STM32L152 features an LCD controller.
- */
-#include <st/l1/stm32l151.dtsi>
+#include <st/l1/stm32l1.dtsi>
```

Comment:
Can you explain a bit more this change?

Reply:
Of course, as I explained in #30790:

It is not needed for it to work, but this is much more in line with the abstraction 'levels' when looking at the other SoC models. If basing it on 151 is preferred, I can change it back, but I would say this is preferred since it might help overcome abstraction issues later on (if things need to be implemented that are specifically for 152, and not 151 etc)

Reply:
So I think it is better to add a separate commit for this change, as it is not needed for the new additions.

Reply:
Done

Reply:
I'm a bit puzzled by this change.
In general we're trying to take advantage of SoC commonalities to avoid code duplication and related issues (maintenance, copy/paste issues, ..). So I don't think we should go this way.

Reply:
In the end, this is your (ST's) call I think, since you're more familiar with the common practices regarding hardware abstraction for the STM32 SoCs. 
Could you discuss this, and let me know if I need to change this back or not?

Reply:
> if things need to be implemented that are specifically for 152, and not 151 

If you compare 151 vs 152, IIRC the only diff is the LCD support. So at the end the L152 should only include L151 + LCD node. So I prefer to revert this and remain consistent across STM32 series.